api/docker-compose.yaml

```diff
@@ -3,6 +3,7 @@ services:
   scripts:
     build:
       context: ./src/dev/server
+    image: allofustest/workbench-dev-api:local
```

Comment:
No - per the PR description, we don't intend to push it anywhere, it's just a tag which will be applied to the image after being built. If an image with this tag already existed on your machine, it would use that instead.